roles/ffh.mesh_wireguard/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+mesh_wireguard_iface: "wg"
+vxlan_iface: "vx"
```

Comment:
maybe call it more precisely vxlan_iface_prefix

Reply:
Done in a62a910